extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -91,53 +139,42 @@ require(['search-tracking', 'uuid', 'wikia.trackingOptIn'], function(searchTrack
 			});
 
 		},
-		trackSearchResultClick: function(clickedElement) {
-			var queryparams = new URL(window.location).searchParams;
-			var query = queryparams.get('search') || queryparams.get('query');
-
-			if (!query) {
-				return;
+		trackSearchResultClick: resultClickTrackerFactory(
+			'article',
+			function (clickedElement) {
+				return clickedElement.getAttribute('data-wiki-id') + '_' + clickedElement.getAttribute('data-page-id');
```

Comment:
```suggestion
				return clickedElement.data('wiki-id') + '_' + clickedElement.data('page-id');
```

Reply:
Would that work? Parameter in a click event's callback is a plain [Node](https://developer.mozilla.org/en-US/docs/Web/API/Node) instance whereas [`.data`](https://api.jquery.com/data/) is a jQuery function that won't work on plain nodes (and isn't about getting data attributes at all, it gets arbitrary data about that node that falls back to `data-*` attribute values if otherwise not found).

Reply:
You're right, it doesn't. :)